src/components/results/shortcircuit/shortcircuit-analysis-one-bus-result.tsx

```diff
@@ -94,13 +96,20 @@ export const ShortCircuitAnalysisOneBusResult = () => {
         []
     );
 
+    const customLabelRowsPerPage = useMemo(() => {
+        return intl.formatMessage({
+            id: 'muiTablePaginationLabelRowsPerPageOneBusSCA',
+        });
+    }, [intl]);
+
     return (
         <ShortCircuitAnalysisResult
             analysisType={ShortCircuitAnalysisType.ONE_BUS}
             analysisStatus={oneBusShortCircuitAnalysisStatus}
             result={result}
             updateResult={updateResult}
             shortCircuitNotif={oneBusShortCircuitNotif}
+            tablePaginationProps={{ labelRowsPerPage: customLabelRowsPerPage }}
```

Comment:
Just give the key here, and let the `CustomTablePagination` translate et forward it to the MUI `TablePagination` component instead of useIntl, useMemo each time

Reply:
Done